examples/landing-zone-v2/tier4/compute-address/Kptfile

```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: example-compute-address
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 tier3 resource example.
```

Comment:
tier4 above - but the depends on at the tier 3 level in the "Example to create..." is ok